python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -1218,8 +1240,10 @@ def describe(self, include=None, exclude=None):
         Examples
         --------
         >>> import cudf
-        >>> gdf = cudf.DataFrame({"Speed": [380.0, 370.0, 24.0, 26.0],
-                                  "Score": [50, 30, 90, 80]})
+        >>> gdf = cudf.DataFrame({
+        ...     "Speed": [380.0, 370.0, 24.0, 26.0],
+        ...      "Score": [50, 30, 90, 80],
+        ... })
         >>> gdf
         Speed  Score
```

Comment:
```suggestion
          Speed  Score
```

While we are at it, can we also indent the column names properly in this file? These are translating to improperly formatted output in the docs, for example: https://docs.rapids.ai/api/cudf/nightly/api_docs/api/cudf.core.groupby.groupby.dataframegroupby.describe

Reply:
Fixed in the areas I spotted (3), let me know if I missed any here.